content/guides/embed/ui-elements/design-system-tokens.md

```diff
@@ -0,0 +1,192 @@
+---
+rank: 12
+related_endpoints: []
+related_guides:
+  - embed/ui-elements
+  - embed/ui-elements/theming-styling
+required_guides:
+  - embed/ui-elements/installation
+related_resources: []
+---
+
+# Design system tokens
+
+This table shows all the design tokens you can use theming and styling purposes. Provided examples and descriptions give clarity and guidance for implementation.
```

Comment:
for theming and styling purposes
Provided examples and descriptions give clarity and guidance for implementation. > this are not examples, this is the currently supported list of design tokens names